main.py

```diff
@@ -50,6 +49,7 @@ def save_user_karma(content, file_path="./user_karma_list.json"):
 
 # load user karma
 user_karma_list = load_user_karma()
+command.command_init(user_karma_list)
 

```

Comment:
Sorry, I forgot this.
```suggestion
    event.event_init(bot)
```